src/inc/til/env.h

```diff
@@ -537,13 +430,21 @@ namespace til // Terminal Implementation Library. Also: "Today I Learned"
         void regenerate()
         {
             // Generally replicates the behavior of shell32!RegenerateUserEnvironment
+            // The difference is that we don't
+            // * handle autoexec.bat (intentionally unsupported).
+            // * call LookupAccountSidW to get the USERNAME/USERDOMAIN variables
+            //   These are available as standard environment variables and they're "volatile"
+            //   (= computed dynamically at each login and then constant until logout).
+            //   We don't expect our process to impersonate another user so we can simplify this.
+            // * call GetComputerNameW to get the COMPUTERNAME variable, for the same reason.

```

Comment:
That's basically what I meant, but I tried writing it in a way that doesn't assume the reader knows what any of this means.

I think `CLIENTNAME` is a bit special in this regard, because it's not stored in the `Volatile Environment` key directly, but rather in it's child key where the key name is your session ID, starting from 1. To me it seems like it is still a constant across an entire session, because every time you login or connect remotely the session ID changes and a new child key is created.

How would you express the comment?